README.md

```diff
@@ -1,22 +1,24 @@
-# [Download the latest release](https://github.com/lwouis/alt-tab-macos/releases/latest)
+# alt-tab-macos
 
-# Purpose
+> Download the latest release [here](https://github.com/lwouis/alt-tab-macos/releases/latest).
+
+## Purpose
```

Comment:
I prefer the original choice of a huge download button here. Also I think the name of the software is already present in the page title (i.e. tab), url, and document title (github title)